### PR TITLE
add new DE translations

### PR DESCRIPTION
**Pull request by RickDBCN**

### CODE REVIEW

**Comment by zepfietje** [comment]
Thanks!

File: packages/forms/resources/lang/de/components.php [hunk 1]
```diff
@@ -16,6 +20,10 @@
                 'label' => 'Löschen',
             ],
 
+            'move_item' => [
+                'label' => 'Verschieben',
+            ],
+
             'move_item_down' => [
                 'label' => 'Runter verschieben',
```
Comment (dissto) [inline packages/forms/resources/lang/de/components.php:28]: @zepfietje 

That is not a proper German translation. If anything, that should be "Nach unten verschieben".

File: packages/forms/resources/lang/de/components.php [hunk 2]
```diff
@@ -204,6 +216,8 @@
 
         'loading_message' => 'Lädt...',
 
+        'max_items_message' => 'Es können nur :count ausgewählt werden.',
+
         'no_search_results_message' => 'Keine Optionen passen zu Ihrer Suche.',
```
Comment (dissto) [inline packages/forms/resources/lang/de/components.php:221]: This also sounds weird. This should be something like: "Die Suche ergab keine Treffer" or "Die Suche lieferte keine Ergebnisse". 🤔

File: packages/forms/resources/lang/de/components.php (hunk 1 above)
Comment (zepfietje) [reply packages/forms/resources/lang/de/components.php:28]: Could you update these strings in a pull request?

**Comment by RickDBCN** [comment]
Hi @dissto,
I fixed these in #4592 

thanks! :)

**Pull request merged**